src/clj_http_ssrf/core.clj

```diff
@@ -32,3 +32,38 @@
           (and hosts (some #(ip/network-contains? %1 host-address) hosts))
           (make-response status headers body)
           :else (client req))))))
+
+(defn wrap-predicates
+  "Accepts a predicate each for requested scheme, url, port and host.
+
+  If any of the predicates return something falsey,
+  clj-http.client/get will return with the specified :status, :headers and :body.
+
+  :status defaults to 403,
+  :headers defaults to {}
+  :body defaults to \"\"
+
+  Returns a map of status code, header and body (as client would return)"
+  [& {:keys [url-pred port-pred host-pred scheme-pred
+             status headers body]
+      :or {status 403 headers {} body ""}}]
+  (fn [client]
+    (fn [req]
+      (let [url (:url req)
```

Comment:
Doesn't require a change but as a personal preference I prefer aligning the bindings vertically like https://github.com/ackerleytng/clj-http-ssrf/blob/ffb4229932eb7aaee3d7a05ac428a54fdc86ff5b/src/clj_http_ssrf/core.clj#L22 since it let bindings more readable.

Reply:
Thanks!

Sure you can edit it, I'll pull.

Do you have a way of automatically aligning it? Emacs?

Reply:
I was trying something with `align-regexp` but I am not able to figure it out. I will edit it, update the changelog, make a release now and update you back. Thanks a lot for the contribution.

Reply:
You could do `M-x align-regexp ( RET` to align them by '(' in let bindings but for complex cases I have to find a way with http://ergoemacs.org/emacs/emacs_align_and_sort.html